frontend/request.go

```diff
@@ -153,9 +158,11 @@ func MaybeSign(ctx context.Context, client gwclient.Client, st llb.State, spec *
 	return signed, nil
 }
 
+const keySkipSigningArg = "DALEC_SKIP_SIGNING"
+
```

Comment:
nit: move this `const` to the top of the file?

Reply:
Done